### ISSUE TITLE
main.js was obfuscated with regular commit

### ISSUE THREAD

**Issue by mariusheise** [SebastianSchultz/ioBroker.enet#42]
Hi,

the main.js was was obfuscated and rendered unusable and non-modifiable with https://github.com/SebastianSchultz/ioBroker.enet/commit/cb3a617cad4c0f78bdb926ca3ade6f5a0636ffb1

Is there a specific reason why this happened? If not, please upload a unobfuscated version...

Thanks,
Marius

**Comment by mtoensing** [comment]
@stoffel7 Can you revert this please?

**Comment by stoffel7** [comment]
Why?

**Comment by mtoensing** [comment]
Is the source available somewhere in the repository as an open-source file? If not, please add the source so we can look at it, fix bugs or use the code to understand this adapter better. What was the reason to obfuscate the file in the first place?

**Comment by stoffel7** [comment]
Hallo, um was geht es, die Sourcen stammen von Sebastian Schulz und mir

**Comment by mtoensing** [comment]
Ja, aber es ist doch ein Open-Source Projekt, oder nicht? Wenn ihr gegen einen Baum fahrt, dann wäre es doch toll, dass jeder das geschriebene weiterverwenden kann. Ich denke, ich muss hier nicht die Vorteile von Open-Source nennen auf GitHub. Aber den Code zu "verschleiern" ist entgegen der Open-Source Idee. Vielleicht war es ja ein Versehen? Deswegen habe ich nach dem Grund gefragt.

**Comment by stoffel7** [comment]
Der Ursprungsadapter für das USB Gateway stammt von Sebastian, ich habe in der main.js den Anteil für den eNet-Server beigetragen, mußte aber ein GHV einhalten, daher die Verschlüsselung, aber ich denke das ist ( aus versch. Gründen) nicht mehr notwendig. Kannst du den Adapter updaten, wenn du die aktuellen Sourcen hättest?

**Comment by mtoensing** [comment]
Den letzten Satz verstehe ich nicht so ganz. Die Funktionsweise ist durch die Verschleierung nicht eingeschränkt. Nutzen kann ich den Adapter. Aber es gibt mit dem aktuellen Stand keine Möglichkeit der Weiterentwicklung. Wenn es nicht mehr nötig ist, dann schieb doch bitte die unverschlüsselten Sourcen hoch. :)

**Comment by stoffel7** [comment]
Wie oben schon geschrieben, der Adapter stammt von Sebastian, ich hab ihm nur die main,js erweitert,
Er hat eigentlich die „Hoheit“ über den Adapter. 
Ich muss es mir mal anschauen…

**Comment by mtoensing** [comment]
Danke. Open-source ohne Open-Source ist irgendwie... Sinnbefreit und am Ende hilft das keinem.

**Comment by sweh** [comment]
Hi. Wenn es nur darum geht, zu verstehen, wie man den eNet-Server anspricht, könnt ihr euch das auch in meinem Homekit-Adapter abschauen. Ist allerdings in Python geschrieben: https://github.com/sweh/enet_homekit

**Comment by mariusheise** [comment]
As I opened the ticket, I also wanted to comment on this again:

What bothered me, was that the obfuscation was done as part of a regular commit merged with multiple other features and bug fixes. The commit message said nothing about an obfuscation.

Further development in the current state is not possible anymore.

**Comment by stoffel7** [comment]
Ich habe jetzt meinen Anteil am  Sourcecode im Original  hochgeladen,  wie schon mehrmals erwähnt, der Adapter stammt von Sebastian. Die Beweggründe für "Verschlüsselung" habe ich oben schon erwähnt.
Ich habe Sebastian angeschrieben, den Adapter upzudaten.
